doc/whats_new.rst

```diff
@@ -22,29 +22,26 @@ Current
 Changelog
 ~~~~~~~~~
 
+- Adding ``TemporalConnectivity``, ``SpectralConnectivity`` and ``SpectroTemporalConnectivity`` as a data structure to hold connectivity data, by `Adam Li`_ (:gh:`6`)
```

Comment:
Single backticks here will link to the objects if you have `default_role = 'py:obj'` in `conf.py`

Reply:
Changed to single ticks.